packages/hawtio/src/plugins/shared/components/JmxContentBeans.tsx

```diff
@@ -0,0 +1,50 @@
+import { useContext } from 'react'
```

Comment:
The file name should be `JmxContentMBean` instead of `JmxContentBean` (note 'M').